packages/flutter_tools/lib/src/web/devfs_web.dart

```diff
@@ -256,18 +247,110 @@ class WebAssetServer {
     }
     return modules;
   }
+
+  // Attempt to resolve `path` to a dart file.
+  File _resolveDartFile(String path) {
+    // If this is a dart file, it must be on the local file system and is
+    // likely coming from a source map request. Attempt to look in the
+    // local filesystem for it, and return a 404 if it is not found. The tool
```

Comment:
Should the comment about 404 move somewhere else? This function cannot return 404.

Reply:
Updated comment